### PR TITLE
Update celery configuration, remove import geonode.messaging to avoid circular dependency

### PR DESCRIPTION
**Pull request by dimasciput**

### CODE REVIEW

**Comment by lucernae** [comment]
If this is problematic, maybe we could just use some hack for now.
Let ```ASYNC_SIGNALS``` be False, then set it to true in bims settings file. This way, we don't have to copy paste geonode settings.
Then we have to load the celery app and queue by ourselves.

**Comment by dimasciput** [comment]
@lucernae can you check again, 

I got all the queues and the tasks loaded now 

```
 -------------- celery@worker v4.1.0 (latentcall)
---- **** -----
--- * ***  * -- Linux-4.9.93-linuxkit-aufs-x86_64-with-debian-9.4 2018-08-03 07:32:01
-- * - **** ---
- ** ---------- [config]
- ** ---------- .> app:         bims:0x7f71aca2a210
- ** ---------- .> transport:   amqp://guest:**@rabbitmq:5672//
- ** ---------- .> results:     disabled://
- *** --- * --- .> concurrency: 4 (prefork)
-- ******* ---- .> task events: ON
--- ***** -----
 -------------- [queues]
                .> all.geoserver    exchange=geonode(topic) key=geoserver.#
                .> broadcast        exchange=geonode(topic) key=#
                .> cleanup          exchange=default(direct) key=cleanup
                .> default          exchange=default(direct) key=default
                .> email            exchange=default(direct) key=email
                .> email.events     exchange=geonode(topic) key=email
                .> geonode          exchange=default(direct) key=geonode
                .> geonode.layer.viewer exchange=geonode(topic) key=geonode.viewer
                .> geoserver.catalog exchange=geonode(topic) key=geoserver.catalog
                .> geoserver.data   exchange=geonode(topic) key=geoserver.catalog
                .> geoserver.events exchange=geonode(topic) key=geonode.geoserver
                .> notifications.events exchange=geonode(topic) key=notifications
                .> update           exchange=default(direct) key=update

[tasks]
  . bims.celery.debug_task
  . geonode.celery_app.debug_task
  . geonode.documents.tasks.create_document_thumbnail
  . geonode.documents.tasks.delete_orphaned_document_files
  . geonode.documents.tasks.delete_orphaned_thumbnails
  . geonode.geoserver.tasks.geoserver_update_layers
  . geonode.layers.tasks.update.delete_layer
  . geonode.maps.tasks.delete_map
  . geonode.services.tasks.harvest_resource
  . geonode.tasks.tasks.send_email
  . geonode.tasks.tasks.send_queued_notifications
  . tasks.update_cluster
  . tasks.update_search_index
```

File: core/settings/geonode_queue_settings.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+from kombu import Exchange, Queue
+
+geoserver_exchange = Exchange("geonode", type="topic", durable=False)
+
+queue_all_events = Queue("broadcast", geoserver_exchange, routing_key="#")
```
Comment (lucernae) [inline core/settings/geonode_queue_settings.py]: You don't have to manually create all this queue. But I think it's ok as long as it doesn't broke.
It's hard to see which one is using which if there are too many queues.

File: core/settings/contrib.py [hunk 2]
```diff
@@ -186,3 +187,65 @@
...
+    },
+    "loggers": {
+        "django": {
+            "handlers": ["console"], "level": "ERROR", },
+        "bims": {
+            "handlers": ["console"], "level": "DEBUG", },
+        "geonode": {
+            "handlers": ["console"], "level": "INFO", },
+        "geonode.qgis_server": {
+            "handlers": ["console"], "level": "ERROR", },
```
Comment (lucernae) [inline core/settings/contrib.py]: put Propagate: False. Because it might show up twice with 'geonode' handler

**Comment by dimasciput** [comment]
Thanks @lucernae , merging this PR now

**Pull request merged**